fixitfriday.ui/src/components/Survey/SurveyVisivilityAdministration/SurveyQuestions.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+import Col from 'react-bootstrap/Col';
+import { Row } from 'react-bootstrap';
+import { SurveyQuestionsPropsType } from './types/SurveyQuestionsPropsType';
+import { SurveyQuestionType } from '../types/SurveyQuestionType';
+import Question from './Question';
+
+const SurveyQuestions: FC<SurveyQuestionsPropsType> = (props: SurveyQuestionsPropsType) => {
```

Comment:
```suggestion
const SurveyQuestions: FC<SurveyQuestionsPropsType> = ({ survey, disabled } : SurveyQuestionsPropsType) => {
```

If we use object destructuring, we could save one line here.